### ISSUE TITLE
Incorrect completion of bupload()

### ISSUE THREAD

**Issue by hexnov** [darkoperator/vscode-language-aggressor#2]
The suggestion recognizes the `bupload()` command while typing, but instead of completing the command to `bupload(beacon, file)`, it inserts `bunlink(beacon, file)`.

The bug is in line 1239 of the cna.json:

```JSON
"bupload": {
    "description": "Ask a Beacon to upload a file",
    "prefix": "bupload",
    "body": [
----->  "bunlink($0${1:beacon}, ${2:file})"
    ]
},
```

**Comment by hexnov** [comment]
The bug was fixed in commit ab0bb4d.

**Comment by darkoperator** [comment]
Forgot to update the issue 🤦‍♂️
